Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_cp.nut

```diff
@@ -426,6 +657,9 @@ void function OnHardpointLeft( entity trigger, entity player )
 		hardpoint.imcCappers.remove( hardpoint.imcCappers.find( player ) )
 	else
 		hardpoint.militiaCappers.remove( hardpoint.militiaCappers.find( player ) )
+	foreach(CP_PlayerStruct playerStruct in file.players)
```

Comment:
you can use .find here

Reply:
no i cant player reference is of type entity and file.players is of type CP_PlayerStruct